plugins/etc/httpd/conf.d/pulp_rpm.conf

```diff
@@ -23,12 +23,21 @@ AddType application/x-x509-ca-cert .crt
 # block rewrites all requests to `/pulp/repos/` to the location of the WSGI
 # application, content.wsgi, provided by the Pulp platform. The content.wsgi
 # application adds support for downloading content on-demand.
+# This block also rewrites requests to show repoview content if it exists.
 <Location /pulp/repos/>
   RewriteEngine On
   RewriteCond %{HTTPS} on
   RewriteRule (.+/pulp/repos/)(.*) /pulp/content/var/www/pub/yum/https/repos/$2 [DPI]
   RewriteCond %{HTTPS} off
   RewriteRule (.+/pulp/repos/)(.*) /pulp/content/var/www/pub/yum/http/repos/$2 [DPI]
+  RewriteCond %{HTTPS} on
```

Comment:
It might be worth separating this section visually from the first section and having the comment about its purpose be right above it. That's a nitpick though and you're welcome to do whatever you want :smile: